aurora/core.py

```diff
@@ -644,14 +1143,26 @@ def run_aurora(
         V_z,
         times_DV=None,
         nz_init=None,
+        ndiv_init=None,
+        npump_init=None,
+        nmainwall_init=None,
+        ndivwall_init=None,
         unstage=True,
         alg_opt=1,
         evolneut=False,
         use_julia=False,
```

Comment:
@antonellozito can you please maintain Julia-backwards-compatibility, even though the more advanced recycling model is not available in Julia? Having some warnings would be nice.